app/services/invitation_links_generator.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class InvitationLinksGenerator
+  def initialize(count)
+    @count = count
+    @links = []
+  end
+
+  def self.call(count)
+    new(count).call
+  end
+
+  def call
+    count.times do
+      invitation = Invitation.create
+      links << link_url(invitation.token)
+    end
+    links
+  end
+
+  private
+
+  attr_accessor :links
+  attr_reader :count, :endpoint
+
+  def link_url(token)
+    "#{ENV['LOCALHOST']}/users/sign_up?token=#{token}"
```

Comment:
https://apidock.com/rails/v4.2.7/ActionDispatch/Http/URL/host

Reply:
http://ruby-doc.org/core-2.4.2/ENV.html#method-c-fetch

Reply:
Sorry, I was mistaken. Sure, we don`t need to use a request for getting a host, but I suggest to set default_url_options once and then get them from application settings, like this: https://www.neontsunami.com/posts/accessing-rails-routes-helpers-from-anywhere-in-your-app.

Reply:
Got your thought, I think it is good idea, resolved